### ISSUE TITLE
pod install 時 cocoapod 報錯 [!] Invalid `fluwx.podspec` file: Unknown alias: xxx.

### ISSUE THREAD

**Issue by dessonchan** [OpenFlutter/fluwx#545]
**Describe the bug**
在 pod install 時 cocoapod 報錯 [!] Invalid `fluwx.podspec` file: Unknown alias: xxx.

**To Reproduce**
1. pubspec.yaml 中有使用 YAML 的 Anchors and Alias 功能
2. pod install
3. cocoa pod 報錯
```
[!] Failed to load 'fluwx' podspec: 
[!] Invalid `fluwx.podspec` file: Unknown alias: xxx.
```

**Comment by JarvanMo** [comment]
可以把你的yaml 里alias贴一下吗？这个问题稍后我修复一下

**Comment by dessonchan** [comment]
```
name: example_app
publish_to: "none"
version: 1.0.0+1

environment:
  sdk: ">=2.17.3 <3.0.0"

dependencies:
  flutter:
    sdk: flutter
  flutter_localizations:
    sdk: flutter
  intl: &intl # Using forked version for multi LocalizationDelegate and fallback support.
    git:
      url: https://github.com/dessonchan/intl.git
      ref: master
  fluwx: ^4.2.4+1

dependency_overrides:
  intl: *intl

fluwx:
  app_id: wx1234567890123
  android:
  ios:
    universal_link: https://example.com
```

用這個 pubspec.yaml 能夠把報錯 reproduce.

```
Analyzing dependencies
[!] Failed to load 'fluwx' podspec: 
[!] Invalid `fluwx.podspec` file: Unknown alias: intl.

 #  from ~/git/example_app/ios/.symlinks/plugins/fluwx/ios/fluwx.podspec:18
 #  -------------------------------------------
 #  flutter_project_dir = calling_dir.slice(0..(calling_dir.index('/ios/.symlinks')))
 >  cfg = YAML.load_file(File.join(flutter_project_dir, 'pubspec.yaml'))
 #  logging_status = "WECHAT_LOGGING=0"
 #  -------------------------------------------
```

 非常感謝！

**Comment by JarvanMo** [comment]
好的，晚上我更新一下

**Comment by JarvanMo** [comment]
Fixed in 4.2.5

**Comment by moonljt521** [comment]
我使用的是 4.2.5 版本，仍然这个错误：
[✓] Flutter (Channel stable, 3.13.9, on macOS 14.4.1 23E224 darwin-arm64, locale zh-Hans-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.1)
[!] Xcode - develop for iOS and macOS (Xcode 15.3)
    ✗ Unable to get list of installed Simulator runtimes.
[✓] Chrome - develop for the web
[✓] Android Studio (version 2022.3)
[✓] IntelliJ IDEA Community Edition (version 2023.1)
[✓] VS Code (version 1.88.0)
[✓] Connected device (3 available)
    ! Device adb-f9ebb660-9IA8Vr._adb-tls-connect._tcp. is offline.
[✓] Network resources


报错：
Analyzing dependencies
5.0.1
[!] Failed to load 'fluwx' podspec: 
[!] Invalid `fluwx.podspec` file: No such file or directory @ rb_sysopen - /Users/moon/Documents/joinu/flutter_mixed/flutter_mixed/.ios/.symlinks/plugins/fluwx/ios/pubspec.yaml.

 #  from /Users/moon/Documents/joinu/flutter_mixed/flutter_mixed/.ios/.symlinks/plugins/fluwx/ios/fluwx.podspec:22
 #  -------------------------------------------
 #  if psych_version_gte_500 == true
 >      cfg = YAML.load_file(File.join(flutter_project_dir, 'pubspec.yaml'), aliases: true)
 #  else
 #  -------------------------------------------
